### PR TITLE
dataflow-task-parallel-library.md: clarified the note

### PR DESCRIPTION
**Pull request by pkulikov**
"The order ... will ... be correctly ordered" reads strange.

### CODE REVIEW

File: docs/standard/parallel-programming/dataflow-task-parallel-library.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ ms.author: "ronpet"
  The default value of <xref:System.Threading.Tasks.Dataflow.ExecutionDataflowBlockOptions.MaxDegreeOfParallelism%2A> is 1, which guarantees that the dataflow block processes one message at a time. Setting this property to a value that is larger than 1 enables the dataflow block to process multiple messages concurrently. Setting this property to <xref:System.Threading.Tasks.Dataflow.DataflowBlockOptions.Unbounded?displayProperty=nameWithType> enables the underlying task scheduler to manage the maximum degree of concurrency.  
   
 > [!IMPORTANT]
->  When you specify a maximum degree of parallelism that is larger than 1, multiple messages are processed simultaneously, and therefore, messages might not be processed in the order in which they are received. The order in which the messages are output from the block will, however, be correctly ordered.  
+>  When you specify a maximum degree of parallelism that is larger than 1, multiple messages are processed simultaneously, and therefore, messages might not be processed in the order in which they are received. The order in which the messages are output from the block is, however, the same one in which the messages are received.
```
Comment (rpetrusha) [inline docs/standard/parallel-programming/dataflow-task-parallel-library.md]: nit unrelated to your PR: no comma after therefore
in which the messages --> in which they

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks good, @pkulikov. I've left two suggestions, one unrelated to your PR.

File: docs/standard/parallel-programming/dataflow-task-parallel-library.md (hunk 1 above)
Comment (pkulikov) [reply docs/standard/parallel-programming/dataflow-task-parallel-library.md]: @rpetrusha thank you for review; I've addressed the feedback.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @pkulikov. I'll merge your PR now.

**Pull request merged**